module/workflow/src/Infrastructure/Persistence/Query/DbalStatusQuery.php

```diff
@@ -109,6 +109,15 @@ public function getStatusCount(Language $translationLanguage, Language $workflow
         return array_values($result);
     }
 
+    public function getMaxIndex(): ?int
```

Comment:
This shode be nullable, there alway sholud be at least one status in system , if not exist that mean data problme